Mathlib/Algebra/Hom/GroupAction.lean

```diff
@@ -517,26 +576,3 @@ theorem comp_id (f : R →+*[M] S) : f.comp (MulSemiringActionHom.id M) = f :=
 #align mul_semiring_action_hom.comp_id MulSemiringActionHom.comp_id
 
 end MulSemiringActionHom
-
-section
-
-variable (M) {R'} (U : Subring R') [IsInvariantSubring M U]
-
-/-- The canonical inclusion from an invariant subring. -/
-def IsInvariantSubring.subtypeHom : U →+*[M] R' :=
-  { U.Subtype with map_smul' := fun m s => rfl }
-#align is_invariant_subring.subtype_hom IsInvariantSubring.subtypeHom
-
-@[simp]
-theorem IsInvariantSubring.coe_subtype_hom : (IsInvariantSubring.subtypeHom M U : U → R') = coe :=
-  rfl
-#align is_invariant_subring.coe_subtype_hom IsInvariantSubring.coe_subtype_hom
-
-@[simp]
-theorem IsInvariantSubring.coe_subtype_hom' :
-    (IsInvariantSubring.subtypeHom M U : U →+* R') = U.Subtype :=
-  rfl
-#align is_invariant_subring.coe_subtype_hom' IsInvariantSubring.coe_subtype_hom'
-
```

Comment:
Why is this not ported?

Reply:
Apparently, this was moved to `algebra.group_ring_action.invariant` in mathlib3 last week, so it's not supposed to be here anymore. Well seen though!